### PR TITLE
Avishay/ocpbugsm 13596

### PR DESCRIPTION
**Pull request by avishayt**
Ran 49 of 49 Specs in 716.120 seconds
SUCCESS! -- 49 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestSubsystem (716.13s)
PASS
ok  	github.com/openshift/assisted-service/subsystem	716.160s

### CODE REVIEW

**Comment by avishayt** [comment]
@mareklibra @jtomasek Please note that this breaks the events API and requires updating the UI.  Instead of fetching events via /events/{entity_id}, you get the events for a cluster (including its hosts) via clusters/{cluster_id}events, and for a specific host pass host_id in query params (clusters/{cluster_id}events?host_id=<host_id>).

File: internal/events/events_api.go [hunk 1]
```diff
@@ -29,16 +30,18 @@ func NewApi(handler Handler, log logrus.FieldLogger) *Api {
 
 func (a *Api) ListEvents(ctx context.Context, params events.ListEventsParams) middleware.Responder {
 	log := logutil.FromContext(ctx, a.log)
-	evs, err := a.handler.GetEvents(params.EntityID.String())
+	var hostID string
```
Comment (ronniel1) [inline internal/events/events_api.go:33]: This should not be used anymore

File: internal/events/events_api.go [hunk 2]
```diff
@@ -29,16 +30,18 @@ func NewApi(handler Handler, log logrus.FieldLogger) *Api {
...
+
+	evs, err := a.handler.GetEvents(params.ClusterID, params.HostID)
 	if err != nil {
-		log.Errorf("failed to get events for id %s ", params.EntityID.String())
-		return events.NewListEventsInternalServerError().
-			WithPayload(common.GenerateInternalFromError(err))
+		log.Errorf("failed to get events for cluster %s host %s", params.ClusterID.String(), hostID)
```
Comment (ronniel1) [inline internal/events/events_api.go:37]: params.HostID.String()

**Pull request merged**